Update descriptions to not discourage GOG players as much (#742)
henpemaz/Rain-Meadow: Workshop description.md

A multiplayer engine, custom game mode, and online arena / story experience for Rain World!

## How it works

> - **You must own the game on Steam to use discoverable lobbies. Without it you can only use "LAN" lobbies.**

Reviewer: ```suggestion
Rain Meadow uses Steam to create and list world-wide discoverable lobbies. 
```

> - It uses Steam to create and list easy-to-join lobbies to connect you to other Rain World players online!

Reviewer: This sentence is wordy and unnecessary. Consider Combining it with the sentence above.
```suggestion

```

> - With or without Steam, you can also create "local"("LAN") lobbies, and join them if you know their IP address.

Reviewer: ```suggestion
- With or without Steam, you can also create LAN lobbies discoverable in your local network. Others can join them through Matchmaking or Direct connection via IP addresses.
```
